src/composables/useSort.js

```diff
@@ -0,0 +1,14 @@
+import { computed } from "vue"
+//export const useSortMovies = (searchedMovies, sortBy)=>computed(()
+export const useSortMovies = (searchedMovies, sortBy) =>
```

Comment:
Improve: `searchedMovies ` and `sortBy ` should be reactive.

Reply:
useSort deleted